hpcm/hpe-mlds-build-hpcm-os-image.md

```diff
@@ -93,18 +99,23 @@ resource_pools:
   - pool_name: aux-pool
     max_aux_containers_per_agent: 100
 EOF
+chmod 640 /etc/determined/master.yaml
 echo "### PRINT MASTER CONFIG START"
 cat /etc/determined/master.yaml
 echo "### PRINT MASTER CONFIG END"
 
 # Run the Master
-docker run -d --restart unless-stopped --name hpe-mlde-master --network host -v /etc/determined/master.yaml:/etc/determined/master.yaml determinedai/hpe-mlde-master:0.17.15
+docker run -d --restart unless-stopped --name hpe-mlde-master \
+  --log-driver journald \
+  --network host \
+  -v /etc/determined/master.yaml:/etc/determined/master.yaml \
+  determinedai/hpe-mlde-master:0.17.15
 ```
 
 And run:
 
 ```bash
-chmod 700 /root/hpe-mlde-master-start.sh
+chmod 750 /root/hpe-mlde-master-start.sh
```

Comment:
It's usually safe to mark files executable files under `/root` as `750` since they're owned by the `root` group by default.

Reply:
Are there any situations where there are members of root's default group who need execute? Least privilege says that the group does not need read and execute, so if there's not a reason to give group access, it should not be done.  I'd suggest leaving it restricted.
```suggestion
chmod 700 /root/hpe-mlde-master-start.sh
```